manifest

```diff
@@ -1,6 +1,5 @@
 #! /bin/bash
 
-export VERSION="32"
```

Comment:
I would prefer to keep the VERSION in this file. The manifest file is meant to hold all the details of the system image. Also, without it, the automated build would not work.